src/android/CameraLauncher.java

```diff
@@ -44,6 +44,7 @@ Licensed to the Apache Software Foundation (ASF) under one
 
 import com.google.gson.Gson;
 import com.google.gson.GsonBuilder;
+import com.thundermaps.android.library.utils.TLog;
```

Comment:
Given that this repository is public, it probably shouldn't have ThunderMaps specific code in it.

Reply:
Could we make the repository private? I do know of any nice way of actually logging stuff to RayGun from this library

Reply:
It doesn't seem very open-sourcey to do that but you'll have to ask Chris.

Reply:
The original plugin was updated but requires a newer version of cordova, so i don't imagine many people would want to use ours

@cloud-source Can you make this repository private

Reply:
This should probably still be made private.

Reply:
Yeah a ticket has been made to do this, (or to do something else?) but for now it has been merged in for release